packages/smooth_app/lib/pages/preferences/user_preferences_dev_mode.dart

```diff
@@ -305,23 +303,10 @@ class UserPreferencesDevMode extends AbstractUserPreferences {
             setState(() {});
           },
         ),
-        ListTile(
-          leading: const Icon(Icons.language),
-          title: DropdownButton<String>(
-            value: userPreferences.appLanguageCode ??
-                Localizations.localeOf(context).toString(),
-            elevation: 16,
-            isExpanded: true,
-            onChanged: (String? languageCode) async =>
-                userPreferences.setAppLanguageCode(languageCode),
-            items: _supportedLanguageCodes.map((Locale locale) {
-              final String localeString = locale.toString();
-              return DropdownMenuItem<String>(
-                value: localeString,
-                child: Text(localeString),
-              );
-            }).toList(),
-          ),
+        LanguagePickerSetting(
```

Comment:
If you make the ui something like this , just title,subtitle and trailing for listtile
![image](https://user-images.githubusercontent.com/57723319/177917385-cd473710-858c-435d-ac03-7dbb139dc1da.png)

Since we are to open this language selection to the end users as well not just in dev

Reply:
Done